### ISSUE TITLE
[Issue]: Fix bug in "Error: No iTunes playables could be found." error/warning message.

### ISSUE THREAD

**Issue by rDamascena** [MichaelYochpaz/iSubRip#55]
### Confirmations

- [X] I have checked the issues tab, and couldn't find an existing open issue for the issue I want to report.

### OS Type

Windows

### Python Version

3.11.6

### Package Version

2.5.1

### Description

Fix bug in "Error: No iTunes playables could be found." error/warning message.

```
Expected:

Error: No iTunes playables could be found.
```

### Output Log

```Text
$ isubrip "https://tv.apple.com/br/movie/bottoms/umc.cmc.4freeyz9nu8443pjd82vplsz8"
User config file detected at 'C:\Users\Adriana\.isubrip\config.toml' and will be used.
Scraping 'https://tv.apple.com/br/movie/bottoms/umc.cmc.4freeyz9nu8443pjd82vplsz8'...
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\Adriana\AppData\Local\Programs\Python\Python311\Scripts\isubrip.exe\__main__.py", line 7, in <module>
  File "C:\Users\Adriana\AppData\Local\Programs\Python\Python311\Lib\site-packages\isubrip\__main__.py", line 166, in main
    scraper_response: ScrapedMediaResponse = scraper.get_data(url=url)
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Adriana\AppData\Local\Programs\Python\Python311\Lib\site-packages\isubrip\scrapers\appletv_scraper.py", line 337, in get_data
    return self.get_movie_data(storefront_id=storefront_id, movie_id=media_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Adriana\AppData\Local\Programs\Python\Python311\Lib\site-packages\isubrip\scrapers\appletv_scraper.py", line 249, in get_movie_data
    raise ScraperError("No iTunes playables could be found.")
isubrip.scrapers.scraper.ScraperError: No iTunes playables could be found.
```

**Comment by MichaelYochpaz** [comment]
Hey @rDamascena, thank you for reporting :)
Found the cause for the issue (all errors & exceptions will show up this way).
I'll release a fix and an update soon.

**Comment by rDamascena** [comment]
Thanks!

**Comment by MichaelYochpaz** [comment]
Fixed on the new [v2.5.2 update](https://github.com/MichaelYochpaz/iSubRip/releases/tag/v2.5.2) :)
